platform/security/idp/security-idp-server/src/test/java/org/codice/ddf/security/idp/server/IdpEndpointTest.java

```diff
@@ -243,6 +243,7 @@ public void setup()
     signature = authNRequestGetSignature;
     signatureAlgorithm = "http://www.w3.org/2000/09/xmldsig#rsa-sha1";
     ssoSAMLResponse = "https://localhost:8993/services/saml/sso?SAMLResponse=";
+    System.setProperty("org.codice.ddf.system.rootContext", "/services");
```

Comment:
Wonder if we should update SystemBaseUrl to use `/services` as the default root context